### PR TITLE
Dependency update

### PR DESCRIPTION
**Pull request by irvanjitsingh**
Bumped dependencies in prep to publish package updates.

`yarn run build` successfully runs.

### CODE REVIEW

**Comment by irvanjitsingh** [comment]
@Gauravjeetsingh I'm not sure if this can be merged yet, since it updates babel and other dependencies, sttm-desktop and any other repos that use anvaad would also need to bump.

File: package.json [hunk 1]
```diff
@@ -1,29 +1,31 @@
 {
   "name": "anvaad-js",
-  "version": "1.4.4",
+  "version": "1.4.3",
```
Comment (Gauravjeetsingh) [inline package.json]: @irvanjitsingh veerji, are we going back on version number for anvaad js?

**Review by Gauravjeetsingh — COMMENTED** [review]
Is the anvaad js version change from 1.4.4 to 1.4.3 in this PR intentional?

**Comment by irvanjitsingh** [comment]
Yes, since 'yarn publish' checks for the current package version and then asks for and bumps it automatically.

File: package.json (hunk 1 above)
Comment (irvanjitsingh) [reply package.json]: No, the current version on npm is 1.4.3, and I bumped it in a prior PR to 1.4.4 but didn't publish a release. The yarn publish flow also handles bumping the version automatically so I made this change to stay in sync with what's on npm currently.

**Review by Gauravjeetsingh — DISMISSED** [review]

**Pull request merged**